src/types/MovieData.ts

```diff
@@ -3,4 +3,5 @@ export interface MovieData {
   Title: string;
   Plot: string;
   imdbID: string;
+  Response: 'True';
```

Comment:
Are you sure that it's the correct type?
Do you really need to hardcore boolean value as string and add a new parameter to MovieData?

Reply:
@polosanya 
I think that I still need to edit type in MovieData / ResponseError because if I check with hasOwn mathod I am getting errors like in attachment
![image](https://user-images.githubusercontent.com/86070599/212003574-93fd7903-63eb-4bba-a275-975a04479f01.png)